lib/features/mailbox_dashboard/domain/usecases/remove_email_cache_on_web.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:tmail_ui_user/features/mailbox_dashboard/domain/repository/email_cache_repository.dart';
+
+class RemoveEmailCacheOnWebInteractor {
+  final EmailCacheRepository emailCacheRepository;
+
+  RemoveEmailCacheOnWebInteractor(this.emailCacheRepository);
+
+  execute() {
```

Comment:
IMO, you should use Either to easily manage return value arguments.

Reply:
I think just get composer cache function need Either, and remove and save no need because function remove and save return void